common/chat/helper.ts

```diff
@@ -166,7 +168,7 @@ const convertConversationInfosToString = (conversationInfos: ConversationInfos):
     };
 
     for (const key in conversationInfos.custom) {
-        if (conversationInfos.custom.hasOwnProperty(key)) {
+        if (Object.prototype.hasOwnProperty.call(key)) {
```

Comment:
WTF, `.call(` is missing the first argument, should be `.call(conversationInfos, key)` ...

Reply:
ahh lost, overlooked this when fixing all the linting stuff. mb

Reply:
didnt seem to fix it :/